static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -32,28 +35,42 @@ type Handlers = {
 const onPlayerStateChangeEvent = (
     event,
     handlers: Handlers,
-    el: HTMLElement
+    el: ?HTMLElement
 ) => {
+    if (el && config.page.isDev) {
```

Comment:
Prefer `config.get('page.isDev')` over direct access to properties on `config`

Reply:
Thanks.  I'll change this too